ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -31,6 +31,10 @@ import { getTransactionTypeTitle } from '../../helpers/utils/transactions.util';
 import ErrorMessage from '../../components/ui/error-message';
 import { toBuffer } from '../../../shared/modules/buffer-utils';
 
+import TransactionDetail from '../../components/app/transaction-detail/transaction-detail.component';
```

Comment:
Why do we use the `.component` postfix for file names, when it's already in the `/components` folder?

Reply:
It is a practice we are moving away from. The reason it exists is to distinguish between "container" components, when we used that pattern for redux. So we have `confirm-transaction-base.component.js` and `confirm-transaction-base.container.js`. This is discouraged for all new components though

Reply:
Thanks for the explanation, great to know!